services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -64,6 +68,16 @@
 
 logger = logging.getLogger(__name__)
 
+DYNAMIC_SIDECAR_SERVICE_EXTENDABLE_SPECS: Final[tuple[list[str], ...]] = (
+    ["labels"],
+    ["task_template", "Resources", "Limits"],
+    ["task_template", "Resources", "Reservation", "MemoryBytes"],
+    ["task_template", "Resources", "Reservation", "NanoCPUs"],
+    ["task_template", "Placement", "Constraints"],
+    ["task_template", "ContainerSpec", "Env"],
+    ["task_template", "Resources", "Reservation", "GenericResources"],
```

Comment:
So the following resources will get overwritten by what is coming in from the webserver and the user's selection:
- `GenericResoruces` like AIRAM, VRAM
- `CPU` both limit and reservation
- `RAM` both limit and reservation